frontend/src/components/Pages/FrontPage/MissionOverview/CreateMissionButton.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Typography, Popover, Icon } from '@equinor/eds-core-react'
+import { Text } from 'components/Contexts/LanguageContext'
+import { Icons } from 'utils/icons'
+import { useRef, useState } from 'react'
+import { useAssetContext } from 'components/Contexts/AssetContext'
+
+export const CreateMissionButton = (): JSX.Element => {
+    const [isPopoverOpen, setIsPopoverOpen] = useState<boolean>(false)
+    const { assetCode } = useAssetContext()
+    const anchorRef = useRef<HTMLButtonElement>(null)
+    const echoURL = 'https://echo.equinor.com/mp?instCode='
+
+    let timer: ReturnType<typeof setTimeout>
+    const openPopover = () => {
+        if (assetCode === '') setIsPopoverOpen(true)
+    }
+
+    const closePopover = () => setIsPopoverOpen(false)
+
+    const handleHover = () => {
+        timer = setTimeout(() => {
+            openPopover()
+        }, 300)
+    }
+
+    const handleClose = () => {
+        clearTimeout(timer)
+        closePopover()
+    }
+
+    return (
+        <>
+            <div onPointerEnter={handleHover} onPointerLeave={handleClose} onFocus={openPopover} onBlur={handleClose}>
+                <Button
+                    onClick={() => {
+                        window.open(echoURL + assetCode)
+                    }}
+                    disabled={assetCode === ''}
```

Comment:
In the "AddMission" button in [ScheduleMissionDialog](https://github.com/equinor/flotilla/blob/3cee004f5ca0938263f9fc55d3e827a73a7cb290/frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog.tsx), the disabled is based on `props.frontPageScheduleButtonDisabled`, should we use something similair here?